redis/redis/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
+    networks: 
+      tut-redis: 
+        ipv4_address: 172.19.0.3
+    volumes:
+    - /var/log/tut-redis/gateway:/kaazing-gateway/log
+  redis:
+    build: broker
+    ports:
+      - "6379:6379"
+    networks: 
+      tut-redis: 
+        ipv4_address: 172.19.0.4      
+    volumes:
+     - /var/log/tut-redis/redis:/data/log/
+networks:
+  tut-redis:
```

Comment:
First of all, I think this is best practices in docker to assign IP addresses.

Aside from that, there was a practical reason for that. I run docker in Windows 7 (boot2docker).

As such, to connect to the actual server I need to create a tunnel in VirtualBox to the machine, and connect to THAT machine, (from Chrome from the Windows 7 host). IF I wouldn't create a static IP, a new IP would be created every time I would tear it down and I would need to enter into VirtualBox GUI and change the tunnel every time.

BTW, related to this, I think more information should be added on the README as to how to run on boot2docker.